Models/global_config.hpp

```diff
@@ -0,0 +1,51 @@
+#include <cstdint>
+
+typedef enum {
+    yaw,
+    pitch,
+    roll,
+    thrust
+} ControlAxis_t;
+
+typedef enum {
+    positive,
+    negative
```

Comment:
Would this be worthwhile? (so you can just multiply by positive or negative to determine motor direction)
```suggestion
    positive = 1,
    negative = -1
```

Alternatively, direction could be replaced with a boolean `inverted` where it is used.

Reply:
I went with the boolean.